src/ai/backend/manager/api/vfolder.py

```diff
@@ -1774,33 +1800,11 @@ async def clone(request: web.Request, params: Any, row: VFolderRow) -> web.Respo
         if 'user' not in allowed_vfolder_types:
             raise InvalidAPIParameters('user vfolder cannot be created in this host')
 
-        source_proxy_name, source_volume_name = root_ctx.storage_manager.split_host(source_folder_host)
-        target_proxy_name, target_volume_name = root_ctx.storage_manager.split_host(target_folder_host)
-        if source_proxy_name != target_proxy_name:
-            raise InvalidAPIParameters('proxy name of source and target vfolders must be equal.')
-
-        # TODO: accept quota as input parameter and pass as argument options
-        try:
-            folder_id = uuid.uuid4()
-            async with root_ctx.storage_manager.request(
-                source_folder_host, 'POST', 'folder/clone',
-                json={
-                    'src_volume': source_volume_name,
-                    'src_vfid': str(row['id']),
-                    'dst_volume': target_volume_name,
-                    'dst_vfid': str(folder_id),
-                },
-                raise_for_status=True,
-            ):
-                pass
-        except aiohttp.ClientResponseError:
-            raise VFolderOperationFailed
-
         user_uuid = str(user_uuid)
         group_uuid = None
         ownership_type = 'user'
         insert_values = {
-            'id': folder_id.hex,
+            'id': uuid.uuid4().hex,
```

Comment:
Where is the original folder ID?
What happens if the folder ID is lost?

Reply:
Thank you for the good point! I thought that UUID and folder_id would always be the same. But, it was actually different and found that other tasks of vfolder had errors. so, I will fix it.